### ISSUE TITLE
Deprecation list for `1.1.0`

### ISSUE THREAD

**Issue by oleksiyskononenko** [h2oai/datatable#3037]
- `dt.isna()`, `dt.abs()`, `dt.exp()`, `dt.log()`, `dt.log10()` are marked internally as [deprecated](https://github.com/h2oai/datatable/blob/main/src/datatable/expr/math.py#L21-L26) for a long time, so they should be replaced with the corresponding `math.*()` functions;
- `dt.len()` should be deprecated as it is replaced by `dt.str.len()`;
-  `dt.frame_columns_virtual()` functionality should be moved to `dt.Frame` class;
- `.stype`, `.stypes`, `.ltype` and `.ltypes` functionality should be accessible through `.type`/`.types`.

**Comment by oleksiyskononenko** [comment]
@st-pasha Do you remember what is the reason to deprecate `dt.isna()` and move it to `dt.math` namespace? Somehow related to not implementing `dt.math.isnan()`?

Since `dt.isna()` could be applied to any column, not just numeric, I don't really understand the logic.

**Comment by st-pasha** [comment]
Could have been by accident. Feel free to un-deprecate it :)

**Comment by oleksiyskononenko** [comment]
Yeah, but then we will have two functions doing the same: `dt.isna()` and `dt.math.isna()`.
